src/Komponents/ButtonCollection/Button.jsx

```diff
@@ -0,0 +1,15 @@
+import { NavLink } from "react-router-dom";
+
+import "./button.css";
```

Comment:
if in folder exist only one file with styles, then i can call it like `styles.css`